R/map_autographr.R

```diff
@@ -55,7 +55,7 @@
 #'   Group variables should have a minimum of 3 nodes,
 #'   if less, number groups will be reduced by
 #'   merging categories with lower counts into one called "other".
-#' @param edge_color Tie variable to be used for coloring the nodes.
+#' @param edge_color Edge variable to be used for coloring the nodes.
```

Comment:
Please revert: while we use edge for graphical elements, users must refer to what is called a tie variable elsewhere in the package.